webapp/src/ts/modals/tour/tour-select.component.ts

```diff
@@ -0,0 +1,39 @@
+import { MmModalAbstract } from '../mm-modal/mm-modal';
+import { BsModalRef } from 'ngx-bootstrap/modal';
+import { Component } from '@angular/core';
+import { TourService } from '@mm-services/tour.service';
+
+@Component({
+  selector: 'tour-select',
+  templateUrl: './tour-select.component.html'
+})
+export class TourSelectComponent extends MmModalAbstract {
+
+  tours: object[];
+
+  constructor(
+    private bsModalRef: BsModalRef,
+    private tourService: TourService,
+  ) {
+    super();
+  }
+
+  ngOnInit() {
+    this.tourService.getTours().then(tours => {
+      this.tours = tours.sort((a: any, b: any) => a.order - b.order);
```

Comment:
I agree that should be returned sorted b the service, I did in the other way just to replicate the old behaviour, where it was sorted in the component, but in the template [here](https://github.com/medic/cht-core/blob/d9ae1aa/webapp/src/templates/modals/tour_select.html#L8):

```html
<a ng-repeat="tour in tourSelectCtrl.tours | orderBy:'order' track by tour.id" class="...
```

Because Angular 2+ does not support sorting in the templates any more, I moved the sorting to the component, but I can move it to the service.